index.html

```diff
@@ -61,15 +61,6 @@
 							<div class="col-md-10 text-right">
 								<ul class="menu">
 									<li><a target="default" class="inner-link" href="#top">Home</a></li>
-									<!-- <li><a target="default" class="inner-link" href="#keynotes">Keynotes</a></li>
```

Comment:
This may be needed later on in the process. I think, we can use this, once the conference organization has more information. Please let the comment stay in there.

Reply:
Generally, I understand why to remove comments but this code is sure to be needed next year. What do you think, should it be removed or could you live with the comment?

Reply:
ohh.. will do it ASAP!!
My bad

Reply:
live with the comments, i have no problems with this..
i will commit again .